salesagility/SuiteCRM
# Fix/module builder subpanel error

Pulled up three functions from
UndeployedMetaDataImplementation.php
to parent class
AbstractMetaDataImplementation.php
so that they would also be available in
UndeployedSubpanelImplementation.php
which needs them because it implements
MetaDataImplementationInterface

## Motivation and Context

Fixes Issue #4431 
Makes #4447 obsolete because it fixes the same issue while avoiding the duplicated code

## How To Test This

In module builder, try to edit a subpanel and make sure no server error occurs and you can edit the subpanel.

![screenshot from 2018-01-09 09-54-15](https://user-images.githubusercontent.com/34056696/34715033-32a92dfe-f523-11e7-9fb3-9996c379f623.png)

## Types of changes


### Final checklist
